packages/data-helper/src/hooks/useNodeInfo/useNodeInfo.js

```diff
@@ -4,18 +4,25 @@ import {getQuery} from './useNodeInfo.gql-queries';
 import {useDeepCompareMemoize} from '../useDeepCompareMemo';
 import {getEncodedPermissionName} from '../../fragments/getPermissionFragment';
 import {getEncodedNodeTypeName} from '../../fragments/getIsNodeTypeFragment';
+import {useSchemaFields} from '../useSchemaFields';
 
 export const useNodeInfo = (variables, options, queryOptions) => {
+    let schemaResult = useSchemaFields();
     // Use ref to avoid infinite loop, as query object will be regenerated every time
     const memoizedVariables = useDeepCompareMemoize(variables);
     const memoizedOptions = useDeepCompareMemoize(options);
-    const {query, generatedVariables, skip} = useMemo(() => getQuery(memoizedVariables, memoizedOptions), [memoizedVariables, memoizedOptions]);
+
+    const {query, generatedVariables, skip, loading} = useMemo(() => getQuery(memoizedVariables, schemaResult, memoizedOptions), [memoizedVariables, memoizedOptions]);
```

Comment:
There's an issue here. The memo will not reexecute once the schema will be loaded, so "loading" and query will never be updated .. at least adds schemaResult in the dependencies array.

Reply:
done